src/pages/using-spark/components/button.mdx

```diff
@@ -19,72 +20,32 @@ Well written button text clearly communicates what action will occur when the us
+- Avoid using too many buttons on a page.
+- Use a [Link](/using-spark/components/link) when the action is to take the user to a new page or to an external site.
 
-#### How to Write Effective Button Text
-- Button text should set clear expectations as to what will happen when activated.
-- Use title case (capitalize the first letter each word).
-- Use task-specific action verbs like: “Add”, “Remove”, “Read Details”, and “Submit”.
-- Use less than 20 characters.
-- Use compact words. For example, instead of “Submit Your Information Now” use “Submit”.
-- Use first-person pronouns. Use “Customize My Loan,” instead of “Customize Your Loan.”
-- Do not use ampersands (&).
-- Do not wrap button text into a second line; it reduces legibility, and starts looking less like an interactable element.
-
-#### Button Styling
-- Text should be center aligned.
-- Buttons should have 16px of padding on the top and bottom, and 24px on the left and right sides.
-- Width should be between 48px and 280px.
-- Grouped buttons should be 16px away from each other.
-- Do not use buttons in paragraphs or within lines of text. Use [Link](/using-spark/components/link) instead.
-
-#### Full Width at Smaller Viewports
-At small viewport sizes, a Button may be full width.
-- Stacked Buttons should have the primary action above secondary actions.
-- Use full width Buttons in containers that take up 1/3 of the grid or smaller.
-
-<SprkDivider
- element="span"
- additionalClasses="sprk-u-Measure"
-></SprkDivider>
+<SprkDivider element="span" additionalClasses="sprk-u-Measure"></SprkDivider>
 
 ## Variants
 
-Use Button variants to emphasize different actions within a page.
+Buttons have different visual weights that help establish a visual hierarchy on a page.
 
-> Variant names imply the order within action hierarchy, but they do not imply the **visual** placement on the page.
+- [Primary Button](#primary-button) (Filled)
+- [Secondary Button](#secondary-button) (Outlined)
+- [Tertiary Button](#tertiary-button) (Text button)
+- [Compact Button](#compact-button)
+- [Full Width Button](#full-width-button)
 
-- [Primary Button](#primary-button)
-- [Secondary Button](#secondary-button)
-- [Tertiary Button](#tertiary-button)
+> Use one primary button as the main call to action to establish the
+> button hierarchy on a screen.
 
 ### Primary Button
 
-Represents the main call to action on the page. Use no more than two Primary Buttons per page.
-
-> When there are multiple buttons, use the Primary Button to indicate the main action.
-
-#### Examples
-- Submit Application
-- Apply to Buy a Home
-- Apply for Refinance
-- In a multi-step form, a Primary Button works with Secondary and Tertiary Button as a set of related actions.
-  - **Primary Button is "Submit"**
-  - Secondary Button is "Back"
-  - Tertiary Button is "Cancel"
+It represents the main action we want users to take on a screen. Filled buttons have the heaviest visual weight.
```

Comment:
Get rid of "it represents"

Change this sentence to not use "we" -- we don't speak in this tense in any of our guidance. We use user-focused writing.

use page instead of "screen" 

 I would combine the first original sentence with the second sentence.